src/screens/rewards/components/RewardsFakeContent.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import Skeleton from '@/components/skeleton/Skeleton';
+import { Box, BoxProps, useForegroundColor } from '@/design-system';
+import { useDimensions } from '@/hooks';
+
+const BORDER_RADIUS = 18;
+
+// Values based off Figma layout
```

Comment:
Will these ever change? Any case where a different phone size or content might change the height of a bit of the UI?

Reply:
also worried about this

Reply:
are these values not design system tokens? they all seem custom which is quirky